docs/insomnia/design-documents.md

```diff
@@ -5,40 +5,54 @@ category: "API Design"
 category-url: api-design
 ---
 
-Design Documents are a way to group and share API specifications, requests and unit tests within Insomnia. For working with collections, see Request Collections.
+Design Documents (also called Documents) are a way to create, group, and share API specifications, requests, and unit tests in Insomnia. 
 
-You can create, delete, and switch between Design Documents and Request Collections from the Dashboard, accessed from the upper left-hand corner of the application.
+Switch between your Documents and Collections via the Dashboard. Inside a Document or Collection, click on **Dashboard**. 
 
-## Create a new Design Document
+{:.alert .alert-primary}
+**Note**: If you don't see a Document or Collection you're looking for, ensure you're in the right [Project](/insomnia/projects) by clicking on the Insomnia dropdown menu.  
+
+![Click Dashboard in the upper left-hand corner to return to a listing of your Documents and Collections.](/assets/images/access-dashboard.png)
+_From inside a Document or Collection, return to your listings by clicking Dashboard._
+
+For information about working with Collections, see [Request Collections](/insomnia/request-collections).
 
-1. Go to the Dashboard
-2. Click "Create" button in upper right hand corner
-3. Select "New Design Document" or a method of importing a
-4. Select how you would like to create a new document.
+## Create a New Design Document
+
+1. Go to the Dashboard.
+2. Click the **Create** button in upper right hand corner.
+3. From the dropdown menu, select **Design Document** or a method of importing.
+    * If you selected **Design Document**, enter a Document name and click **Create**. Spaces and special characters are allowed in your Document name.
+    * If you selected a method of importing, a modal will open prompting you to import your Document. 
 
 {:.alert .alert-primary}
-**Note**: When naming your document, the name selected will be displayed on the Dashboard. Spaces and special characters are allowed. 🤗
+**Note**: Once created, the Document name will be displayed next to the Dashboard navigation link. 
 
 ## Rename a Design Document
 
-1. Go to the Dashboard
-2. Open the action menu (visualized by the three dots) on the document you wish to rename.
-3. Select "Rename" action
-4. Rename the document
+There are two ways to rename a Design Document - from the dashboard and from inside the Design Document. 
+
+### Rename From the Dashboard
+
+1. Open the action menu (visualized by the three dots) on the document you wish to rename.
+2. Select **Rename** action.
+3. Rename the document.
+
+### Rename From Inside the Design Document
```

Comment:
👍🏽 the duplicate and delete steps can also be done from both locations, dashboard and document settings